src/components/Articles.jsx

```diff
@@ -0,0 +1,38 @@
+import { useEffect, useState } from 'react';
+import ArticleCard from './ArticleCard';
+import axios from 'axios';
+
+const Articles = () => {
+	const [articles, setArticles] = useState([]);
+
+	const api = axios.create({
+		baseURL: 'https://nc-news-wbhn.onrender.com/api',
+	});
+
+	const getArticles = async () => {
+		try {
+			const results = await api.get('/articles');
+			setArticles(results.data.articles);
+		} catch (err) {
+			console.log(err);
+		}
+	};
+
```

Comment:
Nice async await. However, your api-calling functions should be in a separate file and imported in for your useEffect